<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateBranchDialog</class>
 <widget class="QDialog" name="CreateBranchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QListWidget" name="variableListWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>141</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VARIABLE 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>VARIABLE 2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Variable </string>
   </property>
  </widget>
  <widget class="QPushButton" name="variable_1Button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OPERATOR</string>
   </property>
  </widget>
  <widget class="QComboBox" name="conditionComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>IF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WHILE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operator</string>
   </property>
  </widget>
  <widget class="QListWidget" name="operatorListWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>141</width>
     <height>241</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Equal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NotEqual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Lesser</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Greater</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LesserOrEqual</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GreaterOrEqual</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="operatorButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="variable_2Button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="variableNUmberButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateBranchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateBranchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
